src/skimage/restoration/_rolling_ball.py

```diff
@@ -43,6 +72,12 @@ def rolling_ball(
         default value; typically equal to the maximum number of virtual cores.
         Note: This is an upper limit to the number of threads. The exact number
         is determined by the system's OpenMP library.
+    downscale_2D : bool, optional
```

Comment:
I'm wondering, if we can make this API more general. For example using something like
```
downscale : {"2d", "none"}, optional
```
with "none" being the default would allow for other kinds of optimizations. Curious what you think.